src/Common/nunit/CommandLineOptions.cs

```diff
@@ -66,9 +75,9 @@ public CommandLineOptions(params string[] args)
                 Parse(args);
         }
         
-        #endregion
+#endregion
         
-        #region Properties
+#region Properties
```

Comment:
All the regions got unindented. This happens with VS2015 whenever you add new `#if` statements. Should we just accept the changes until VS gets its act together?